client/js/components/statement/listOriginalStatements/ListOriginalStatements.vue

```diff
@@ -0,0 +1,565 @@
+<template>
+  <div>
+    <dp-map-modal
+      ref="mapModal"
+      :procedure-id="procedureId" />
+
+    <dp-loading
+      v-if="isLoading"
+      class="u-mt"/>
+
+    <template v-else>
+      <dp-pager
+        v-if="pagination.currentPage && originalStatements.length > 0"
+        :class="{ 'invisible': isLoading }"
+        class="mt-4 mb-2"
+        :current-page="pagination.currentPage"
+        :total-pages="pagination.totalPages"
+        :total-items="pagination.total"
+        :per-page="pagination.perPage"
+        :limits="pagination.limits"
+        @page-change="fetchOriginalStatementsByPage"
+        @size-change="handleSizeChange"
+        :key="`pager1_${pagination.currentPage}_${pagination.perPage}`" />
```

Comment:
If there is time you could order the attributes in this file

Reply:
Thanks, done in 0de3ec324c8bda003f564d73af019dfb1dc260b2